en/05_Contributing/01_Code.md

```diff
@@ -6,245 +6,283 @@ icon: code
+
+Following these additional guidelines for your pull request will improve the chances that your change will be merged:
+
+- Link to [the relevant GitHub issue](#make-or-find-a-github-issue) from your pull request description
+- Link to your pull request from that issue (in the description if you can edit it, or in a comment otherwise)
+- Explain your implementation. If there's anything which you needed to do a deep dive to find the best way to do it, or anything potentially controversial, etc, you can add a comment to your own pull request explaining what you did and why you did it that way.
+
+### Step 6: Recieve and respond to feedback {#recieve-feedback}
+
+Once your pull request is created, it's not the end of the road.
+
+#### Automated feedback
+
+Most of the core and supported repositories have an automated GitHub Actions workflow which will run on your pull request. When it finishes running, check for any failed builds.
+
+If you think a build has failed for reasons unrelated to the changes you've made, point that out in a comment. If the failure _is_ related to your changes, then make any adjustments necessary to resolve the problems.
+
+#### Peer review feedback
+
+The core team will review the pull request as time permits. They will most likely have some questions for you and may ask you to make some changes, so make sure you have [configured your GitHub notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications) appropriately.
+
+- Try to respond to feedback in a timely manner. PRs that go for a while without a response from the author are considered stale, and will be politely chased up. If a response still isn't received, the PR will eventually be closed.
+- If you don't agree with a requested change, provide a clear reason why. Bonus points for showing precedent in the existing codebase. But be open to accepting alternative view points - if a member of the core team insists that you make the change after responding to your reasoning, it's often best to defer to their judgment.
+
+#### Resolving merge conflicts
+
+If other changes are merged in before yours, your pull request may end up with merge conflicts. You'll need to resolve those by rebasing your branch on top of the target branch, and then manually resolving the merge conflicts.
+
+[warning]
+Using `--force-with-lease` is necessary after a rebase, because otherwise GitHub will reject your push. This is because your commit hashes will have changed. But beware that you are explicitly telling GitHub that you are intentionally overriding data. MAke sure you have the correct branch name when doing that step to avoid accidentally overriding other branches in your forked repository.
```

Comment:
```suggestion
Using `--force-with-lease` is necessary after a rebase, because otherwise GitHub will reject your push. This is because your commit hashes will have changed. But beware that you are explicitly telling GitHub that you are intentionally overriding data. Make sure you have the correct branch name when doing that step to avoid accidentally overriding other branches in your forked repository.
```

Reply:
Done